include/openssl/evp.h

```diff
@@ -344,6 +344,7 @@ int (*EVP_CIPHER_meth_get_ctrl(const EVP_CIPHER *cipher))(EVP_CIPHER_CTX *,
 # define         EVP_CTRL_SET_PIPELINE_INPUT_BUFS        0x23
 /* Set the input buffer lengths to use for a pipelined operation */
 # define         EVP_CTRL_SET_PIPELINE_INPUT_LENS        0x24
+# define         EVP_CTRL_SET_DRBG                       0x25
```

Comment:
I suggest adding a getter as well, for use with implementations that won't or can't include `evp_locl.h`

Reply:
As in an implementation of an EVP_CIPHER_CTX that's not part of OpenSSL? Is that supported by something like an engine?

Reply:
@levitte: I'm waiting for your reply

Reply:
I'm not sure I understand the full implication of your question.  A getter control would retrieve the associated DRBG, regardless of if the content of the `EVP_CIPHER_CTX` is "part of OpenSSL" or not, no?

Reply:
The question is if there are cases where it's not part of OpenSSL. I really don't like adding APIs that are never going to get used.

Reply:
It's of course about a cipher implementation, not the ctx than can be external. I'll add it.